x-pack/plugins/security_solution/public/timelines/components/fields_browser/index.test.tsx

```diff
@@ -54,13 +41,11 @@ describe('StatefulFieldsBrowser', () => {
           timelineId={timelineId}
           toggleColumn={jest.fn()}
           width={FIELD_BROWSER_WIDTH}
-          removeColumn={removeColumnMock}
-          upsertColumn={upsertColumnMock}
         />
       </TestProviders>
     );
 
-    expect(wrapper.find('[data-test-subj="show-field-browser"]').first().text()).toEqual('Columns');
+    expect(wrapper.find('[data-test-subj="show-field-browser-gear"]').exists()).toBe(true);
```

Comment:
nit: At first I thought the addition of `-gear` here in `show-field-browser-gear` was a typo, because the fields browser `EuiButtonIcon` has an `iconType="list"`, but then realized this naming is related to the history of the icon, which changed over time.

When the fields browser was first introduced, the icon used to represent it was a gear. It was later changed from a gear to the list icon.

Consider removing the `-gear` from `data-test-subj="show-field-browser-gear"` in `x-pack/plugins/security_solution/public/timelines/components/fields_browser/index.tsx` to avoid confusion, since `Customize event renderers` uses the gear now.